spec.md

```diff
@@ -1213,11 +1213,10 @@ wire w: UInt
 w <= mux(c, b, a)
 ```
 
-In the case where an invalid statement is followed by a conditional statement
-containing a connect to the invalidated component, the resulting connection to
-the component can be expressed using a conditionally valid expression. See
-[@sec:conditionally-valids] for more details about the conditionally valid
-expression.
+Because invalid statements assign indeterminate values to components, a FIRRTL
+Compiler is free to choose any value for an indeterminate value when resolving
```

Comment:
"any **concrete** value" ?

Reply:
Good point, are you fine with use of _specific_ value (which is the language in the indeterminate value section)?